### ISSUE TITLE
Suboptimal reproduced results of MEND on MMEdit

### ISSUE THREAD

**Issue by Wang-Z-W** [zjunlp/EasyEdit#593]
Thank you for your awesome work!

However, I encountered the same problem of #186 . The reproduced results of MEND on MMEdit seem to be suboptimal compared to yout reported results. The following are my reproduced results after editing BLIP-2 OPT by MEND on 100 VQA samples:
rewrite_acc: 0.07
rephrase_acc: 0.06
rephrase_image_acc: 0.06
locality_acc: 1.0
multimodal_locality_acc: 1.0

I used the following functions to reproduce the results:
multimodal_edit.train_MEND_Blip2OPT_VQA
multimodal_edit.edit_MEND_Blip2OPT_VQA
and maintained the original code without any modification. Note that the function MultimodalEditor.edit_dataset is used in multimodal_edit.edit_MEND_Blip2OPT_VQA, and it does not support sequential editing. But after several edits, the logits are NAN. Since the experiment is conducted when sequential_edit=False, I have no idea why this kind of thing happen.

**Comment by Wang-Z-W** [comment]
Thank you for your suggestion! I will give it a try next. If possible, could you kindly provide the specific hyperparameters used in the MEND experiment from your paper? I would be extremely grateful!

**Comment by zxlzr** [comment]
This paper was published two years ago, so the YAML file has likely changed, and I can't guarantee that the provided configuration parameters are optimal. I suggest you try tuning them yourself. Feel free to reach out if you have any questions.

**Comment by Wang-Z-W** [comment]
Thank you! I'll have a try.
